### DISCUSSION TITLE
Can't make change in my profile name

### DISCUSSION THREAD

**Discussion by Maxmillium** [community/community#111056] [New to GitHub]
Hello: New to GitHub, must have made a profile sometime ago because I'm logged in.
Instructions for profile name change begin with:
"In the upper-right corner of any page, click your profile photo, then click Settings."
I don't HAVE a profile photo, or anything similar.  How do i get to "Settings"?

**Comment by thekarthikr** [comment]
Go to this URL: [github.com/settings/profile](https://github.com/settings/profile). It'll ask you to sign in if you aren't logged in.

**Comment by koeppel** [comment]
"In the upper-right corner of any page, click your profile photo, then click Settings." you indeed have a default profile photo, the round avatar that is displayed next to your name.

**Comment by DanielFreitassc** [comment]
Even without a profile photo, you can still access your settings on GitHub. Here's how you can do it:

1. Go to the GitHub homepage (github.com) and ensure you're logged in.
2. Look at the upper-right corner of the page. You should see your profile icon or a silhouette. Click on it.
3. A dropdown menu should appear. Look for the "Settings" option and click on it.

This should take you to your account settings where you can make changes, including changing your profile name. Let me know if you need further assistance!
